libraries/standard/http/cbmc/proofs/HTTPClient_Send/HTTPClient_Send_harness.c

```diff
@@ -30,8 +30,8 @@
 
 void harness()
 {
-    HTTPRequestHeaders_t * pRequestHeaders = NULL;
-    HTTPResponse_t * pResponse = NULL;
+    HTTPRequestHeaders_t * pRequestHeaders;
+    HTTPResponse_t * pResponse;
 
     /* Ideally, this should be allocated in the heap but doing so makes CBMC
```

Comment:
Where does this comment apply? Gaurav noticed it first and we want to know if it is still relevant?

Reply:
You are right. It no longer applies, but we used to allocate it on the stack. Not sure why it works when transport interface is defined in another header file.